File: javascript/iniciante/1019.js
const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
});

rl.on("line", (line) => {
  const N = parseInt(line.trim());
  let restante = N;

  const horas = Math.floor(restante / 3600);
  restante = restante % 3600;
  const minutos = Math.floor(restante / 60);
  const segundos = restante % 60;

  console.log(`${horas}:${minutos}:${segundos}`);
  rl.close();
});

File: swift/iniciante/1019.swift
import Foundation

if let input = readLine(), let N = Int(input) {
    var restante = N

    let horas = restante / 3600
    restante = restante % 3600
    let minutos = restante / 60
    let segundos = restante % 60

    print("\(horas):\(minutos):\(segundos)")
}
